flaggr/src/main/java/com/comuto/flag/strategy/Strategy.java

```diff
@@ -16,5 +16,5 @@
      * @param context The context used to do operation
      * @return true if the flag is activated, false otherwise
      */
-    boolean isFlagActivated(Flag flag, FlagContextInterface context);
```

Comment:
Should we expose FlagResultStatus to the clients of flaggr? Or is it an implementation detail?